unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -372,23 +382,38 @@ protected void HideAllAgentRenderers()
                                 break;
                             }
                         }
+
+                        //If CapsuleCast check failed, move on to next direction from p (otherwise Check 2's error message could return a false-positive)
+                        if(shouldEnqueue == false) {
```

Comment:
Two things here:

1. Can you make this conditional check `!shouldEnqueue` instead of `== false`?
2. Can you add a space after `if`? (This is also a C# styling convention.)

So it goes from
```c#
if(shouldEnqueue == false) {
```
to
```c#
if (!shouldEnqueue) {
```

Reply:
Done.